public/static/js/projects.js

```diff
@@ -3,6 +3,8 @@ mciModule.controller('ProjectCtrl', function ($scope, $window, $http, $location,
   $scope.availableTriggers = $window.availableTriggers
   $scope.userId = $window.user.Id;
   $scope.isAdmin = $window.isAdmin;
+  //hasViewPermissions is true if the user has permissions to view at least one project
+  $scope.hasProjectViewPermissions = $window.allTrackedProjects.length > 0; 
```

Comment:
I would avoid declaring this and check trackedProjects.length > 0 where you need to use it. Having fewer variables in $scope makes the data model easier to think about and also means angular doesn't need to check as many things for updates